### PR TITLE
Fix keyClk Descriptions

### PR DESCRIPTION
**Pull request by adosikas**
keyClk() was flat out wrong, keyClk(e) ommited -1 return value

### CODE REVIEW

**Comment by oldmud0** [comment]
Generally speaking, I've always thought the E2Helper descriptions to be very unhelpful or even nonexistent. I suggest we take a pass over them.

**Comment by CaptainPRICE** [comment]
@oldmud0 Well it is easy to find undocumented functions. However, it is harder to make sure all of the E2 function descriptions are correct, in both way, grammar and their purpose.. and consistent.
I guess we could check for spelling errors in e2descriptions via Travis-CI, but I think that'd be just a small part of it.
Anyways, I believe I had created an issue for this in the past, actually it is just about a year back. #1185

File: lua/wire/client/e2descriptions.lua [hunk 1]
```diff
@@ -310,8 +310,8 @@ E2Helper.Descriptions["boxCenterW(e:)"] = "Same as using E:toWorld(E:boxCenter()
 E2Helper.Descriptions["aabbMin(e:)"] = "Returns the entity's (min) axis-aligned bounding box"
 E2Helper.Descriptions["aabbMax(e:)"] = "Returns the entity's (max) axis-aligned bounding box"
 E2Helper.Descriptions["aabbSize(e:)"] = "Returns the entity's axis-aligned bounding box size"
-E2Helper.Descriptions["keyClk()"] = "Returns 1 if the E2 was triggered by owner pressing/releasing key"
-E2Helper.Descriptions["keyClk(e)"] = "Returns 1 if the E2 was triggered by player pressing/releasing key"
+E2Helper.Descriptions["keyClk()"] = "Returns the player that pressed/released the key if the E2 was triggered by runOnKey"
```
Comment (CaptainPRICE) [inline lua/wire/client/e2descriptions.lua]: There is a missing `s` in `runOnKey`.

File: lua/wire/client/e2descriptions.lua [hunk 2]
```diff
@@ -310,8 +310,8 @@ E2Helper.Descriptions["boxCenterW(e:)"] = "Same as using E:toWorld(E:boxCenter()
...
+E2Helper.Descriptions["keyClk(e)"] = "Returns 1 or -1 if the E2 was triggered by the player pressing or releasing key"
```
Comment (CaptainPRICE) [inline lua/wire/client/e2descriptions.lua]: Maybe correct this. Something like "Returns 1 if a player pressed or -1 if released a key."

**Pull request merged**